### PR TITLE
Remove the exportPublic + ACL-based approach to exported object

### PR DESCRIPTION
**Pull request by prb112**
expiry #2054

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: docs/src/pages/guides/FHIRServerUsersGuide.md
Comment (lmsurpre) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:2069]: looks like we had a documentation bug here.  the enableParquet descriptiion was on the exportPublic line and vice-versa.
mind copying the current `exportPublic` description down to the enableParquet line?

Comment (prb112) [inline docs/src/pages/guides/FHIRServerUsersGuide.md:2069]:
```suggestion
|`fhirServer/bulkdata/storageProviders/<source>/enableParquet`|boolean|Whether or not the server is configured to support export to parquet; to properly enable it the administrator must first make spark and stocator available to the fhir-bulkdata-webapp (e.g through the shared lib at `wlp/user/shared/resources/lib`)|
```

**Review by lmsurpre — APPROVED** [review]
A bit nervous about deleting this function in a patch release, without any real warning for folks that might be relying on it.  However, I think we can cover it in the "special migration considerations" and presigned urls should be the way to go.

**Comment by prb112** [comment]
I think we can also hold this until after 4.8.1

**Comment by lmsurpre** [comment]
Maybe for 4.8.1 we do a docs-only change that says this setting is deprecated.  Then we merge this after 4.8.1?

**Comment by prb112** [comment]
> Maybe for 4.8.1 we do a docs-only change that says this setting is deprecated. Then we merge this after 4.8.1?

https://github.com/IBM/FHIR/pull/2383 updates the docs

**Pull request merged**
